tests/ImageSharp.Tests/TestImages.cs

```diff
@@ -25,6 +25,9 @@ public static class Png
             public const string Powerpoint = "Png/pp.png";
             public const string SplashInterlaced = "Png/splash-interlaced.png";
             public const string Interlaced = "Png/interlaced.png";
+            public const string Palette8Bpp = "Png/Palette-8bpp.png";
```

Comment:
fixed in https://github.com/SixLabors/ImageSharp/pull/292/commits/d6a2d40a80a60e0eb25861d7714f4cfd5b8cc90d